packages/components/src/base-control/stories/index.js

```diff
@@ -8,7 +8,7 @@ import { boolean, text } from '@storybook/addon-knobs';
  */
 import BaseControl from '../';
 
-export default { title: 'Components|Base-control', component: BaseControl };
+export default { title: 'Components|Base Control', component: BaseControl };
```

Comment:
@fernandovbs Awesome! One final tiny nitpick ⛏ (apologies!). I think it should be `BaseControl` (no space). Just like the React component name :).

Thank you!

Reply:
No problem :smile:. Done!